about.css

```diff
@@ -0,0 +1,790 @@
+    color: #fff;
+  }
+
+  .d-menu .d-m a {
+    font-size: 16px;
+    font-family: 'Lato', sans-serif;
+    text-decoration: none;
+    margin-left: 10%;
+    padding: 4%;
+    color: #fff;
+    white-space: nowrap;
+  }
+
+  /* .d-menu .c {
+        margin-left: -5px;
+    } */
+  .mobile-only,
+  .modal-container {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: #fff;
+    width: 100%;
+  }
+
+  .d-m-container {
+    width: 100%;
+    margin-left: 60%;
+    padding: 2%;
+  }
+
+  .desktop-m {
+    text-decoration: none;
+    margin-right: 10px;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    color: #000;
+    padding: 20px;
+    filter: drop-shadow(0 0 16px rgba(255, 255, 255, 0.24));
+  }
+
+  .desktop-m:hover {
+    color: #ec5242;
+  }
+
+  .header-intro {
+    margin-top: 6.56%;
+  }
+
+  .p-1 {
+    display: none;
+  }
+
+  .p-2 {
+    font-size: 16px;
+    font-family: 'Lato', sans-serif;
+    line-height: 1.4;
+    font-weight: normal;
+    text-align: center;
+    width: 60%;
+    margin-left: 20%;
+  }
+
+  .h-1,
+  .h-3 {
+    font-size: 62px;
+    font-weight: bolder;
+    font-family: 'Lato', sans-serif;
+    text-align: center;
+    margin-left: 7%;
+    line-height: 0.4;
+  }
+
+  .h-1 {
+    margin-top: 10%;
+  }
+
+  hr {
+    margin-top: -70px;
+  }
+
+  .h-1-act,
+  .h-1-part {
+    font-size: 18px;
+    color: #000;
+    font-weight: bold;
+    text-align: center;
+    font-family: 'Lato', sans-serif;
+    margin-left: 5%;
+    margin-top: 3%;
+  }
+
+  .p-1-act {
+    font-size: 14px;
+    font-weight: normal;
+    color: #000;
+    text-align: center;
+    font-family: 'Lato', sans-serif;
+    margin-left: 5%;
+    margin-bottom: 0;
+  }
+
+  .expo-1 {
+    margin-bottom: 10%;
+    border: 1px solid rgba(64, 83, 252, 0.24);
+    margin-left: 33.33%;
+    width: 250px;
+    height: 70px;
+    filter: drop-shadow(0 0 16px rgba(64, 83, 252, 0.24));
+  }
+
+  .expo-1 img {
+    width: 280px;
+    height: 100px;
+    margin-left: -48%;
+    margin-top: -52%;
+    padding: 100px;
+    border-radius: 5px;
+  }
+
+  .p-img {
+    font-size: 14px;
+    font-family: 'Lato', sans-serif;
+    margin: -30% 0 0 0;
+    text-align: center;
+    border-top: 1px solid grey;
+    border-bottom: 1px solid grey;
+    width: 100%;
+    padding: 2%;
+    white-space: nowrap;
+  }
+
+  #main-1 {
+    display: none;
+  }
+
+  #main-0 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+  }
+
+  .main-program {
+    width: 100%;
+    height: 100%;
+  }
+
+  .m-program {
+    display: grid;
+    grid-template-columns: repeat(6, 19%);
+    grid-template-rows: 100%;
+    padding: 3%;
+    grid-gap: 0;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    margin-left: 10%;
+    height: 100%;
+    border: 0;
+  }
```

Comment:
- [ ] Card items are not properly aligned to the center as in the design. Please consider making some adjustments.

| **The provided design** |
| --- |
| ![card-design](https://user-images.githubusercontent.com/45002654/133834539-eaaf1030-6db5-440e-8d5b-e6ee0b2339d2.PNG) |

| **Your Implementation** |
| --- |
| ![card](https://user-images.githubusercontent.com/45002654/133834287-d0a13ee9-5cc4-4c85-a1b0-14944e21a5f7.PNG) |